bika/lims/browser/bootstrap/static/css/bootstrap-integration.css

```diff
@@ -364,3 +364,8 @@ https://github.com/senaite/senaite.lims/pull/92 */
 #content > h1:first-of-type {
     margin-top: 0;
 }
+
+/* Do not display Clients top-folder in nav-bar when client */
+body.userrole-client div.navbar li.section-clients {
+  display:none;
+}
```

Comment:
Can we remove the `View` permission for clients on this folder?
I would like to avoid such CSS visibility hacks if possible...

Reply:
I've done it already. But still, when client navigates to i.e. a Sample, the navbar renders then the Clients folder again, even though he/she does not have "View" permission.

Reply:
See https://github.com/senaite/senaite.core/pull/1573/files#diff-453afd17fc06f6552688aa474b8c6cc5